.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Package**
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+Applicable Panoptes resource IDs (project, workflow, etc) to demonstrate the issue:
```

Comment:
If this is aimed at project owners or volunteers, then I'd avoid mentioning resource IDs.

Reply:
Anyone can report on github, but most of our bug reports from project owners or volunteers come through contact email or on Talk, so I'm ok with this being here.